st2common/packaging/debian/postinst

```diff
@@ -3,6 +3,9 @@
 if [ -d /etc/st2 ]; then
   sed -i -r "s~(st2.*)/conf~/etc/\1~g" /etc/st2/st2.conf
   sed -i "s~vagrant~/home/stanley~g" /etc/st2/st2.conf
+
+  # Enable auth by default
+  sed -i "s~enable = False~enable = True~g" /etc/st2/st2.conf
```

Comment:
Moved to two configs approach in 99ca964fd8dee5eb73389ef8b625921b472f5eba

I can also move tests config to `conf/st2.tests.conf` if ppl think this is a better place.